### PR TITLE
User Documentation: Mealplanner UI - Login, Logout, Mealplans

### PR DESCRIPTION
**Pull request by MohamedHAlsadek**
**Describe the technical changes contained in this PR**
Be clear about purpose and scope of the change

**Previous behaviour**
Describe the product behaviour before this change, if relevant

**New behaviour**
Describe how product behaviour is altered by this change

**Related issues addressed by this PR**
List issue numbers using the "Fixes #xxx" syntax

**Have the following been addressed?**
- [ ] Have test cases been created for all of the changes?
- [ ] Do all of the test cases pass?
- [ ] Has the testing been done using the default docker-compose environment?
- [ ] Are documentation changes required?
- [ ] Does this change break or alter existing behaviour?

### CODE REVIEW

File: docs/2-Logout.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+**Usage**
+
+1. **Logging Out**
+
+    - Ensure you are logged in to your account
+    - Locate the (Logout) button to the right of the top of the page, beside the account name
+    - Click on the (Logout) button to initiate the logout process
+![Logout Page](https://snipboard.io/lMLQaj.jpg)
```
Comment (rorybray) [inline docs/2-Logout.md]: Please check images in to the docs/ directory alongside the .md files

File: docs/1-Login..md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+1. **Accessing the Login Page**
+    - Navigate to the GoodMealPlan website by typing the URL [https://goodmealplan.com/](https://goodmealplan.com/) in the web browser's address bar
+    - Click on the "LOGIN" button in the login window to access your account page
+
+2. **Logging In**
+    - Enter your registered email address in the "Email" field
+    - Enter your password in the "Password" field
+    - Click on the (LOGIN) button to proceed
+![Login Page](https://snipboard.io/bVYejn.jpg)
```
Comment (rorybray) [inline docs/1-Login..md]: Please check images in to the docs/ directory alongside the .md files

File: docs/1-Login..md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#Login To Mealplanner 
+
+###Overview
+The login page allows registered users to sign in to their accounts to access personalized features such as saved meal plans and  favorite recipes.
+
+**Usage**
+
+1. **Accessing the Login Page**
+    - Navigate to the GoodMealPlan website by typing the URL [https://goodmealplan.com/](https://goodmealplan.com/) in the web browser's address bar
+    - Click on the "LOGIN" button in the login window to access your account page
```
Comment (rorybray) [inline docs/1-Login..md]: we can replace this with a brief description of what to expect. Clicking doesn't come until the next step.

File: docs/3-Creat a meal Plan.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    - If you're not already logged in, sign in to your account
+    - Once logged in, you'll be directed to the meal plans page
+
+2. **Create a New Meal Plan**
+    - On the meal plans page, click on the "Create Meal Plan" button
+![Login Page](https://snipboard.io/klgLQe.jpg)
+*Screenshot: Create Meal Plan*
+         
+3. **Enter Meal Plan Information**
+    - Select "Create New Meal Plan"
```
Comment (rorybray) [inline docs/3-Creat a meal Plan.md:20]: This doesn't mention the assign user drop-down or start date

**Review by rorybray — CHANGES_REQUESTED** [review]
Just a couple of changes needed

File: docs/2-Logout.md (hunk 1 above)
Comment (MohamedHAlsadek) [reply docs/2-Logout.md]: Done

File: docs/1-Login..md (hunk 2 above)
Comment (MohamedHAlsadek) [reply docs/1-Login..md]: Done

File: docs/1-Login..md (hunk 3 above)
Comment (MohamedHAlsadek) [reply docs/1-Login..md]: Done

File: docs/3-Creat a meal Plan.md (hunk 4 above)
Comment (MohamedHAlsadek) [reply docs/3-Creat a meal Plan.md:20]: Done

**Review by rorybray — CHANGES_REQUESTED** [review]
One small thing I didn't notice before. Could you rename the file from 1-Login..md to 1-Login.md (take out the extra dot)

**Review by MohamedHAlsadek — COMMENTED** [review]
Done. Thanks, Rory.

**Review by rorybray — APPROVED** [review]
LGTM

**Pull request merged**